src/test/util_sds_check/util_sds_check.h

```diff
@@ -0,0 +1,58 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/* Copyright 2025, Hewlett Packard Enterprise Development LP */
+
+/*
+ * util_sds_check.h -- header for unit test
+ */
+
+#include <stdlib.h>
+#include <stdbool.h>
+#include "unittest.h"
+#include "shutdown_state.h"
+#include "set.h"
+
+int os_part_deep_common(struct pool_replica *rep, unsigned partidx, void *addr,
+	size_t len, int flush);
+/* This function does not take part in this unit test. */
+int
+os_part_deep_common(struct pool_replica *rep, unsigned partidx, void *addr,
+	size_t len, int flush)
+{
+	return 0;
+}
+
+#define DIRTY_SET 1
+#define DIRTY_CLEAR 0
+
+/* Invalid value for all test cases */
+#define INVALID_VALUE 0
+
+#define CHECKSUM 0
+#define UUID 1
+#define USC 1
+
+#define SDS_CUSTOM(USC, UUID, DIRTY_BIT, CHECKSUM) \
```

Comment:
```suggestion
#define SDS_CUSTOM(USC, UUID, DIRTY_BIT) \
```

I would argue neither `CHECKSUM` nor `INVALID_VALUE` are exactly a checksum nor an invalid one. They are just unrelated. 😆
I think you should drop this argument altogether and when it is necessary to have a valid checksum for `Pool_sds` just calculate it as follows:

```c
shutdown_state_checksum(&Pool_sds, &Rep);
```

Reply:
Still waiting for this one to happen. 🙏

Reply:
Done.